patroni/validator.py

```diff
@@ -18,6 +18,10 @@
 from .utils import data_directory_is_empty, get_major_version, parse_int, split_host_port
 
 
+# Additional parameters to fine-tune validation process
+VALIDATION_PARAMS : Dict[str, bool] = dict()
```

Comment:
Variable with all capital letters in its name is treated by `pyright` as a constant, which is not entirely true for this specific case.
```suggestion
_validation_params : Dict[str, Any] = {}
```

Besides that, I would prefer to have a function that will modify global variable.